src/main/java/com/mosaicchurchaustin/oms/client/GroupMeClient.java

```diff
@@ -0,0 +1,42 @@
+package com.mosaicchurchaustin.oms.client;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+
+@Component
+public class GroupMeClient {
+
+    private static final String GROUPME_BOT_POST_URL = "https://api.groupme.com/v3/bots/post";
+
+    @Value("${groupme.botId}")
+    private String botId;
+
+    public void postMessage(String message) throws Exception {
```

Comment:
If we get a message ID or some other detail back, it might be good to return it.  
We may also want to persist it.  My thinking is if we're going to be integrating with GroupMe, we may also end up doing the same for assigning orders. So, if we're able to react to the original message when someone in the new system assigns it to themselves, we'd need to know the message id attached to the order. 
Not sure if that's feasible, but definitlely not possible if we don't keep track of the message id